engine/gamesys/src/gamesys/components/comp_gui.cpp

```diff
@@ -660,6 +660,11 @@ namespace dmGameSystem
 
     static void OnWindowResizeCallback(const dmGui::HScene scene, uint32_t width, uint32_t height)
     {
+        dmRender::HDisplayProfiles display_profiles = (dmRender::HDisplayProfiles) dmGui::GetDisplayProfiles(scene);
+        if (display_profiles && !dmRender::GetAutoLayoutSelection(display_profiles))
```

Comment:
Can display_profiles really be null?

Reply:
we can't communicate `gui`<->`gamesys` another way

Reply:
Wrong comment?